deploy/requirements.txt

```diff
@@ -20,3 +20,4 @@ django-session-security==2.6.5
 django-phonenumber-field==3.0.1
 phonenumberslite==8.10.16
 coa-django-countable_field==1.6
+django-flags==4.2.4
```

Comment:
not sure if https://github.com/cityofaustin/techstack/issues/3078 has acceptance criteria, should figure out what that is so we can get that QA'd.